publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -303,6 +310,14 @@ public ResponseEntity generateAndPublish(final String msgProtocol, final String
         if (bodyJson.isJsonObject()) {
             events.add(getAsJsonObject(bodyJson));
         } else if (bodyJson.isJsonArray()) {
+            //here add check for limitation for events in array is fetched from REMReM property and checked during publishing.
+            if (bodyJson.getAsJsonArray().size() > maxSizeOfInputArray) {
+                return createResponseEntity(HttpStatus.BAD_REQUEST, JSON_ERROR_STATUS,
+                        "The number of events in the input array is exceeded the allowed limit of 250 events. " +
+                                "This issue occurred because the input array contains more than 250 events, which is not supported by the system. "
+                                + "To resolve this, please divide the events in to smaller arrays, ensuring each array contains no more than 250 events,"
+                                + "and try to publish them again. This limitation helps to maintain system performance and stability.");
+            }
             for (JsonElement element : bodyJson.getAsJsonArray()) {
```

Comment:
Why to call getAsJsonArray() multiple time?

Reply:
fix it